### PR TITLE
Using code tabs to simplify instructions for log collection

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Use code tabs to simplify log collection instructions.

### Motivation

Customer feedback.

### Preview link

* https://docs-staging.datadoghq.com/gus/log-collection-update/logs/log_collection/?tab=tailexistingfiles#custom-log-collection

### CODE REVIEW

File: content/logs/log_collection/_index.md [hunk 1]
```diff
@@ -48,20 +48,34 @@ If an integration does not support logs by default, use the custom file configur
...
+2. Create a new `conf.yaml` file in this new repository.
+3. Add a custom log collection configuration with the following parameter:
+
+| Parameter        | Presence  | Description                                                                                                                                                                                                                                                                                                                |
+| ------           | -------   | ------                                                                                                                                                                                                                                                                                                                     |
+| `type`           | mandatory | type of log input source (**tcp** / **udp** / **file**)                                                                                                                                                                                                                                                                    |
```
Comment (NBParis) [inline content/logs/log_collection/_index.md]: we should add: `windows_event`, `docker`, `journald`

File: content/logs/log_collection/_index.md [hunk 2]
```diff
@@ -48,20 +48,34 @@ If an integration does not support logs by default, use the custom file configur
...
+| `source`         | mandatory | Attribute that defines which integration is sending the logs. If the logs do not come from an existing integration then this field may include a custom source name. But we recommend matching this value to the namespace of any related [custom metrics][2] you are collecting, e.g, `myapp` from `myapp.request.count`) |
+| `sourcecategory` | optional  | Multiple value attribute. Can be used to refine the source attribtue. Example: `source:mongodb, sourcecategory:db_slow_logs`                                                                                                                                                                                               |
+| `tags`           | optional  | Add tags to each log collected, [learn more about tagging][13].                                                                                                                                                                                                                                                            |
+
```
Comment (NBParis) [inline content/logs/log_collection/_index.md]: New parameter have been added for the above new type:

For journald: `include_units` and `exclude_units`: optional, Include or exclude a specific list of journald units

For windows_event: `channel_path`, Mandatory, specify the list of windows event channel to collect.

File: content/logs/log_collection/_index.md [hunk 3]
```diff
@@ -48,20 +48,37 @@ If an integration does not support logs by default, use the custom file configur
 
 ## Custom log collection
 
-The Datadog Agent v6 can collect logs from files or the network (TCP or UDP) and forward them to Datadog. To configure this, create a new repository and yaml file named after your log source  in the Agent's **conf.d** directory ( `conf.d/python.d/conf.yaml` for python logs, ...) and set these options:
+The Datadog Agent v6 can collect logs from files or the network (TCP or UDP) and forward them to Datadog:
```
[the remark this answers is not in the document]
Comment (l0k0ms) [reply content/logs/log_collection/_index.md]: Across the doc we use `The Datadog Agent`.

[the remark this answers is not in the document]
Comment (phrawzty) [reply content/logs/log_collection/_index.md]: It's true that we say "The Agent" and "The Datadog Agent" all over, but "The Datadog Agent **v6**" comes across oddly.

I *guess* the first two phrases address an object, so the article is correct - but `v6` is a complement, which affects a subject, so using the article is incorrect.

**Pull request merged**